src/registrar/templatetags/custom_filters.py

```diff
@@ -174,3 +177,65 @@ def has_contact_info(user):
 @register.filter
 def model_name_lowercase(instance):
     return instance.__class__.__name__.lower()
+
+
+@register.filter(name="is_domain_subpage")
+def is_domain_subpage(path):
+    """Checks if the given page is a subpage of domains.
+    Takes a path name, like '/domains/'."""
+    # Since our pages aren't unified under a common path, we need this approach for now.
+    url_names = [
```

Comment:
Don't think this is necessary to address in this PR, but I imagine this may get bigger as we introduce pages/subpages so should we consider a ticket to revisit this and refactor our paths in a way that's easier to recognize a page's more meta traits? @abroddrick